src/components/message-list/_macro.spec.js

```diff
@@ -20,6 +20,7 @@ const EXAMPLE_MESSAGE_LIST_MINIMAL = {
             },
             fromText: 'Example Sender 1',
             dateText: 'Tue 4 Jul 2020 at 7:47',
+            bodyLabel: 'Body',
```

Comment:
This param shouldn't be at the message level, it should be at the top param level like the other label params